pkg/minikube/machine/cache_images.go

```diff
@@ -269,8 +269,15 @@ func transferAndLoadImage(cr command.Runner, k8s config.KubernetesConfig, src st
 		return errors.Wrap(err, "runtime")
 	}
 
-	if err := r.RemoveImage(imgName); err != nil {
-		klog.Warningf("remove image: %v", err)
+	exists, err := imageExists(r, imgName)
```

Comment:
I worry this add performance downside to it, how about instead of Image List, we do something cheaper ? try to remove it, and if the error message says it doesnt exist then only klog.Info if error message is not that klog.Warnning

or alternatively we could do

klog.Warniningf("failed to remove image (which might be okay if it doesn't exists")